pgvectorscale/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "vectorscale"
-version = "0.4.0"
+version = "0.5.0"
```

Comment:
I think it's better to have a separate PR (e.g. #140 for 0.4.0) for updating the version.